### ISSUE TITLE
Is cross-language training supported? For example, training a model with Japanese speech and make it speak English

### ISSUE THREAD

**Issue by DogeLord081** [neonbjb/tortoise-tts#442]

**Comment by Keith-Hon** [comment]
I have finetuned a model for Cantonese, a tonal language, like Mandarin. I also tried to mix it with english. Here is the result.

https://soundcloud.com/keith-hon/cantonese-english-mixed-tts-with-voice-clone

**Comment by DogeLord081** [comment]
> Totally possible, I have finetuned a model for Cantonese, a tonal language, like Mandarin. I also tried to mix it with english. Here is the result.
> 
> https://soundcloud.com/keith-hon/cantonese-english-mixed-tts-with-voice-clone
> 
> https://github.com/neonbjb/tortoise-tts/issues/493

How? Can you tell me what model you used and do you mind sharing the training script? Thanks

**Comment by Keith-Hon** [comment]
> > Totally possible, I have finetuned a model for Cantonese, a tonal language, like Mandarin. I also tried to mix it with english. Here is the result.
> > https://soundcloud.com/keith-hon/cantonese-english-mixed-tts-with-voice-clone
> > #493
> 
> How? Can you tell me what model you used and do you mind sharing the training script? Thanks

I used DLAS and changed the vocab before finetune

**Comment by superxii** [comment]
@Keith-Hon 

Hi, I am planning to finetune a model for Cantonese also. Would you mine to share some tips on the finetuning? Like the training samples sizes and the way to evaluate?

**Comment by Keith-Hon** [comment]
> @Keith-Hon
> 
> Hi, I am planning to finetune a model for Cantonese also. Would you mine to share some tips on the finetuning? Like the training samples sizes and the way to evaluate?

i dun have the way to evaluate except my ear at the moment. sample sizes depend on your expected accuracy.

**Comment by Keith-Hon** [comment]
@superxii do you fine tune it for a company? Or a side hustle?

**Comment by superxii** [comment]
This is for my personal interest only, inspired by 允光 ai. But I will use company GPU to train.

**Comment by 1879687161** [comment]
@Keith-Hon Hi, do you mind making a repo or instruction about how you finetune your model? I am finetuning mine with Chinese, but it was terrible. If you could provide any ideas, that will be really helpful. Thank you!

**Comment by Keith-Hon** [comment]
> @Keith-Hon Hi, do you mind making a repo or instruction about how you finetune your model? I am finetuning mine with Chinese, but it was terrible. If you could provide any ideas, that will be really helpful. Thank you!

https://github.com/neonbjb/tortoise-tts/discussions/129#discussioncomment-3157890

James talked about that earlier

**Comment by Keith-Hon** [comment]
> This is for my personal interest only, inspired by 允光 ai. But I will use company GPU to train.

you might check out mrq repo, people shared some insights there as well

https://git.ecker.tech/mrq/ai-voice-cloning/issues/152

**Comment by superxii** [comment]
Really appreciate of your sharing and your demo. Keep it up!!!!!
